### DISCUSSION TITLE
Display start time of event instead of date

### DISCUSSION THREAD

**Discussion by b-lor** [MMRIZE/MMM-CalendarExt3#107] [Q&A] [answered]
I'd like to display the start time of an event rather than the date.  Here's a screenshot of what it looks like now.
![2023-11-04 19 07 39](https://github.com/MMRIZE/MMM-CalendarExt3/assets/20853239/d6e72574-d7e7-4b9c-8cfc-78490d56695f)

Here's a screenshot of what I'd like it to look like. 
![2023-11-04 18 43 48](https://github.com/MMRIZE/MMM-CalendarExt3/assets/20853239/808b3232-c939-4314-8eae-0db9f079ec4c)

Even swap, it helps to know when an event starts.  Awesome project.  

Thank you in advance.

**Comment by eouia** [comment]
By default,  without misconfiguration, it will show the start time. What did you do in the config?

**Reply by eouia** [reply, marked as answer]
You probably set “eventTimeOptions” by wrong value.

**Reply by b-lor** [reply]
Yes, that was it.  I mis-read the description while trying to adjust some settings.  Thank you so much
